components/gauges/arc/overview.md

```diff
@@ -12,25 +12,15 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/arc-gauge" target="_blank">Telerik Arc Gauge for Blazor</a> represents [numerical values]({%slug arc-gauge-pointers%}) on an arc [scale]({%slug arc-gauge-scale%}).
 
-#### This article is separated in the following sections: 
-
-* [Basics](#basics)
-
-* [Features](#features)
-
-* [Methods](#methods)
-
-## Basics
-
->caption To add a Telerik Arc Gauge for Blazor to your application:
+## Creating Blazor Arc Gauge
 
 1. Add the `<TelerikArcGauge>` tag.
 
 1. Add an instance of the `<ArcGaugePointer>` to the `<ArcGaugePointers>` collection.
 
 1. Provide a `Value` for the `<ArcGaugePointer>`.
 
-1. Add a [Center Label Template]({%slug arc-gauge-labels%}#center-template) (optional)
+1. (optional) Add a [Center Label Template]({%slug arc-gauge-labels%}#center-template)
 
 >caption Basic Telerik Arc Gauge for Blazor.
```

Comment:
We are removing all unnecessary screenshots, except those, which:
- demonstrate some user interaction via animation
- include annotations and embedded explanations